### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by poojasabnani**

### CODE REVIEW

File: alert.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          </div>
+          <div class="iframe-container">
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=night-owl&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=29px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-primary%2522%253EThis%2520is%2520primary%2520alert%21%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-secondary%2522%253EThis%2520is%2520secondary%2520alert%21%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-success%2522%253EThis%2520is%2520success%2520alert%21%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-warning%2522%253EThis%2520is%2520warning%2520alert%21%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520alert-failure%2522%253EThis%2520is%2520failure%2520alert%21%253C%252Fdiv%253E"
+  style="width: 732px; height: 240px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (DanishOnGit) [inline alert.html]: Minor suggestion: we can avoid the inline style here and in other places

File: avatar.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        </ul>
+      </aside>
+      <main class="main ">
+        <h1 class="heading-primary">Avatar</h1>
+
+        <p class="heading-primary fs1">
+          You can select avatars with image based on size:
+        </p>
+        <div class="main-content">
+          <div class="component-container  pd2">
```
Comment (DanishOnGit) [inline avatar.html]: Hi Pooja,
Class names (fs1, pd2) can be changed to something more descriptive

File: button.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+          </div>
+          <h3 class="heading-primary">Icon buttons</h3>
+    <div class="component-container">
+        <button class="button button-icon fa fa-home fa-2x"></button>
+        <button class="button button-icon fa fa-bars fa-2x"></button>
+        <button class="button button-icon fa fa-github fa-2x"></button>
+        <button class="button button-icon fa fa-linkedin fa-2x"></button>
+</div>
```
Comment (DanishOnGit) [inline button.html]: code formatting can be done here

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+@import url('stylesheets/card.css');
+@import url('stylesheets/input.css');
+@import url('stylesheets/lists.css');
+@import url('stylesheets/navigation.css');
+@import url('stylesheets/modal.css');
+@import url('stylesheets/rating.css');
+@import url('stylesheets/slider.css');
+@import url('stylesheets/grid.css');
+@import url('stylesheets/text-utilities.css');
+@import url('stylesheets/toast.css');
```
Comment (DanishOnGit) [inline styles.css]: minor refactor: you can rearrange the imports alphabetically

File: stylesheets/badge.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    color: white;
+    border-radius: 50%;
+    min-width: 0.7rem;
+    height: 0.7rem;
+    border: 3px white solid;
+}
+
+.fa-bell {
+    color: rgb(182, 182, 182);
+}
```
Comment (DanishOnGit) [inline stylesheets/badge.css]: color variable can be used

File: stylesheets/badge.css [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+.badge-icon-container {
+    position: relative;
+    
+}
+
+.icon-badge {
+    position: absolute;
+    background-color: rgb(236, 84, 14);
```
Comment (DanishOnGit) [inline stylesheets/badge.css]: color variable can be used

File: stylesheets/input.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    padding: 0.5rem 0.5rem;
+}
+
+.input-textbox:focus{
+    border: 2px solid var(--primary-color);
+}
+
+.input-textbox-error:focus{
+    border:2px solid rgb(248, 22, 22);
+}
```
Comment (DanishOnGit) [inline stylesheets/input.css]: color variable here too

**Review by DanishOnGit — COMMENTED** [review]
Hey Pooja,
Its quite cleanly written. I have suggested mostly around class names and css variables.

**Comment by sarthaklav** [comment]
You are repeating some part of code in almost every html file. like buttons in sidebar this -
     ` </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./alert.html">Alert</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./badge.html">Badge</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./button.html">Button</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./card.html">Card</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./image.html">Images</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./input.html">Input</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./list.html">Lists</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./navigation.html">Navigation</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./modal.html">Modal</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./rating.html">Rating</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./utilities.html">Text utilities</a>
          </li>
          <li class="navbar-vertical-item">
            <a class="aside-link" href="./toast.html">Toast</a>
          </li>`
          
          Here you can do one thing - make a separate file and keep this code in that like we do for header and footer.
          Rest is very commendable work.

**Comment by poojasabnani** [comment]
@sarthak1209 hey, on each page the active component of sidebar is different hence the sidebar code is same in each file except the focus component. Let me know if any suggestions on how can I keep this code in one place but ensure that active component is also styled if clicked.

**Review by sarthaklav — COMMENTED** [review]
Hey Pooja, 
You can change the background of the header to make it look uniform. 

Rest is good. The code is well organised and clean.

File: alert.html (hunk 1 above)
Comment (poojasabnani) [reply alert.html]: done

File: avatar.html (hunk 2 above)
Comment (poojasabnani) [reply avatar.html]: done

File: button.html (hunk 3 above)
Comment (poojasabnani) [reply button.html]: done

File: styles.css (hunk 4 above)
Comment (poojasabnani) [reply styles.css]: done

File: stylesheets/badge.css (hunk 5 above)
Comment (poojasabnani) [reply stylesheets/badge.css]: done

File: stylesheets/badge.css (hunk 6 above)
Comment (poojasabnani) [reply stylesheets/badge.css]: done

File: stylesheets/input.css (hunk 7 above)
Comment (poojasabnani) [reply stylesheets/input.css]: done

**Comment by poojasabnani** [comment]
@sarthak1209 : changed the header

**Comment by poojasabnani** [comment]
@sarthak1209 @DanishOnGit Addressed your comments. Let me know if we are good to go to merge the PR

**Comment by DanishOnGit** [comment]
Hi @py93 , Looks good to me!!

**Comment by sarthak1209** [comment]
> @sarthak1209 @DanishOnGit Addressed your comments. Let me know if we are good to go to merge the PR
> @sarthak1209 : changed the header

Yes!! looks good @py93

**Comment by sarthaklav** [comment]
> @sarthak1209 : changed the header



> @sarthak1209 @DanishOnGit Addressed your comments. Let me know if we are good to go to merge the PR

Yes!! looks good @py93

**Review by gkanishk — COMMENTED** [review]
@py93 few suggestions:
- Add hover/focus effect to buttons
- Second add some padding top left/right inside you card that looks bit weird also try to center image/content and see if that looks good.
Screenshot 2021-05-22 at 1 43 17 PM

rest LGTM!

**Pull request merged**